### ISSUE TITLE
[Feature]: Visualization of scene of point clouds, meshes, and other 3d shapes over time

### ISSUE THREAD

**Issue by etaoxing** [wandb/client#3591]
### Description

Based on the latest docs, wandb currently supports 3D visualizations for the these filetypes: `['obj', 'gltf', 'glb', 'babylon', 'stl', 'pts.json']`, as well as point clouds given as `nx3, nx4, nx6`.

It'd be nice to have a convenient way to do 3D scene visualizations over time (rather than a point cloud or mesh at a single instance in time).  

### Suggested Solution

Would like an API similar to:

```python
scene = wandb.3DScene()

timesteps = ...

# sequence of point clouds
pcd_sequence = ...

# sequence of meshes
vertices_sequence = ...
faces_sequence = ...

for i, t in enumerate(timesteps):
    # probably need another parameter for length, assuming inputs are padded
    scene.add_pointcloud(t=t, pcd_sequence[i,...])
    scene.add_mesh(t=t, v=vertices_sequence[i,...], f=faces_sequence[i,...])
```

Would also be nice to be able to log batches of 3d object sequences. 

### Alternatives

I've found that [meshplot](https://skoch9.github.io/meshplot/tutorial/) is the easiest for HTML visualization of meshes / point clouds. 

However, [kaolin](https://kaolin.readthedocs.io/en/latest/notes/checkpoints.html) functionality is closer to the feature request, as they use [USD format](https://kaolin.readthedocs.io/en/latest/modules/kaolin.visualize.html#kaolin.visualize.Timelapse.add_pointcloud_batch) to store the underlying scene over time.

### Additional Context

Related issue https://github.com/wandb/client/issues/3023#issue-1077225817 on optimizing 3d visualizations.

**Comment by nate-wandb** [comment]
Hi @etaoxing, thank you for the future request along with a few possible solutions. 

The request seems pretty straight forward but could you explain "Would also be nice to be able to log batches of 3d object sequences" a little? Are you asking for the ability to pass in several 3d object sequences at once rather than having to iterate over your data and pass in each individual sequence into `wandb.3DScene()`?

Thank you,
Nate

**Comment by etaoxing** [comment]
> Are you asking for the ability to pass in several 3d object sequences at once rather than having to iterate over your data and pass in each individual sequence into wandb.3DScene()?

Yes, if I'm generating multiple sequences of 3d objects, it'd be nice to add them to the scene together, and have a way to track which object is which. Possibly:
```python
scene.add_object("object_name", timesteps, object_data_seq)
# or
scene.add_objects(["object1", "object2"], timesteps, [object1_data_seq, object2_data_seq])
```

**Comment by nate-wandb** [comment]
@etaoxing Thank you for the clarifications! I'll pass this on to our engineering team and follow up if they have any additional questions or if they are able to implement these asks in the product.

**Comment by nate-wandb** [comment]
Hi @etaoxing, I wanted to see if this solution would work for you in this case:

If you upload a 3D object multiple times to the same name it will actually show up in the UI with a step slider where you could scroll through each step. 

So in this example you could iterate through each "frame" in your 3D scene and log it to the same name with `wandb.log()` and then use the step slider to visualize it over time in the UI.

**Comment by nate-wandb** [comment]
Hi @etaoxing, thank you for the additional context. I'll put in this feature request with our engineering team and follow up with you if this gets selected for development.
